include/clap/clap.h

```diff
@@ -56,8 +56,10 @@
 #include "ext/draft/file-reference.h"
 #include "ext/draft/midi-mappings.h"
 #include "ext/draft/preset-load.h"
-#include "ext/draft/quick-controls.h"
+#include "ext/draft/remote-controls.h"
 #include "ext/draft/state-context.h"
 #include "ext/draft/surround.h"
 #include "ext/draft/track-info.h"
```

Comment:
```suggestion
#include "ext/draft/track-info.h"
#include "ext/draft/triggers.h"
```
triggers extension needs to be added as well